CheckDrive.Mobile/CheckDrive.Mobile/CheckDrive.Mobile/ViewModels/Operator/OperatorHomeViewModel.cs

```diff
@@ -0,0 +1,79 @@
+﻿using CheckDrive.Mobile.Models;
+using CheckDrive.Mobile.Models.Review;
+using CheckDrive.Mobile.Stores.Operator;
+using CheckDrive.Mobile.Views.Operator;
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Threading.Tasks;
+using Xamarin.Forms;
+
+namespace CheckDrive.Mobile.ViewModels.Operator
+{
+    public class OperatorHomeViewModel : BaseViewModel
+    {
+        private readonly IOperatorStore _operatorStore;
+        public List<OilMark> OilMarks { get; set; }
+        public ObservableCollection<DriverDto> Drivers { get; set; }
+        public Command<DriverDto> ShowReviewPopupCommand { get; }
+        public DateTime CurrentDate { get; }
```

Comment:
look at the other ViewModels to see how the properties and fields are declared. I mean the spaces and the orders in which they're put.